crowd_anki/export/anki_exporter.py

```diff
@@ -93,3 +93,18 @@ def _copy_media(self, deck, deck_directory):
                             str(media_directory.resolve()))
             except IOError as ioerror:
                 print("Failed to copy a file {}. Full error: {}".format(file_src, ioerror))
+
+    def _clear_media(self, media_directory: Path):
+        """Clear existing media from export dir before copying.
+
+        Try to be minimally invasive to avoid catastrophic data
+        removal if the relevant variables become incorrectly set
+        elsewhere.
+
+        """
+        unsafe_dirs = [Path.home()]
```

Comment:
Why end the paranoia there? 😁 include '/' and '' too!

Reply:
Thanks so much for reviewing/approving! :)

That was the initial version, but had thought that if somebody is using Anki as root then either:

1. they're running in docker, for testing (?), so likely don't have anything too valuable at "/"
2. they like living on the edge.

(Unlike HOME it also doesn't help on Windows, where I have no idea what the appropriate paths would be.)

OTOH it doesn't hurt, so I've added it back in! :)